app/src/main/java/com/uc/mymovies/MainActivity.kt

```diff
@@ -1,61 +1,61 @@
-        testMovieData.add(movie1);
-        testMovieData.add(movie2);
-        testMovieData.add(movie3);
-        testMovieData.add(movie4);
-        testMovieData.add(movie5);
-        testMovieData.add(movie6);
-        testMovieData.add(movie7);
-        testMovieData.add(movie8);
-        testMovieData.add(movie9);
-        testMovieData.add(movie10);
-
-        findViewById<ListView>(R.id.list_view).adapter = ListAdapter(main_activity, testMovieData);
-
-        var searchButton = findViewById<ImageButton>(R.id.searchButton);
-
-        var searchFilter = findViewById<EditText>(R.id.searchFilter);
+        var movie1: Movie = Movie("Avatar", "Fantasy", "Jack Black", "https://upload.wikimedia.org/wikipedia/en/thumb/b/b0/Avatar-Teaser-Poster.jpg/220px-Avatar-Teaser-Poster.jpg", "A movie about a blue person.")
+        var movie2: Movie = Movie("Joker", "Fantasy", "Jack Black", "https://m.media-amazon.com/images/M/MV5BNGVjNWI4ZGUtNzE0MS00YTJmLWE0ZDctN2ZiYTk2YmI3NTYyXkEyXkFqcGdeQXVyMTkxNjUyNQ@@._V1_.jpg", "A movie about a blue person.")
+        var movie3: Movie = Movie("Spiderman", "Fantasy", "Jack Black", "https://upload.wikimedia.org/wikipedia/en/f/f9/Spider-Man_Homecoming_poster.jpg", "A movie about a blue person.",)
+        var movie4: Movie = Movie("Thor", "Fantasy", "Jack Black", "https://images-na.ssl-images-amazon.com/images/I/91P1wWqX63L._SL1500_.jpg", "A movie about a blue person.")
+        var movie5: Movie = Movie("Avengers", "Fantasy", "Jack Black", "https://upload.wikimedia.org/wikipedia/en/8/8a/The_Avengers_%282012_film%29_poster.jpg", "A movie about a blue person.",)
+        var movie6: Movie = Movie("Pokemon", "Fantasy", "Jack Black", "https://m.media-amazon.com/images/M/MV5BN2NkZjA0OWUtZDgyMy00MjIxLWJhZTEtYjdmYzVjZTQ3YWRiL2ltYWdlL2ltYWdlXkEyXkFqcGdeQXVyMzM4MjM0Nzg@._V1_UY1200_CR85,0,630,1200_AL_.jpg", "A movie about a blue person.")
+        var movie7: Movie = Movie("Iron Man", "Fantasy", "Jack Black", "https://upload.wikimedia.org/wikipedia/en/0/00/Iron_Man_poster.jpg", "A movie about a blue person.")
+        var movie8: Movie = Movie("The Boondock Saints", "Fantasy", "Jack Black", "https://m.media-amazon.com/images/M/MV5BYzVmMTdjOTYtOTJkYS00ZTg2LWExNTgtNzA1N2Y0MDgwYWFhXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_.jpg", "A movie about a blue person.")
+        var movie9: Movie = Movie("Training Day", "Fantasy", "Jack Black", "https://images-na.ssl-images-amazon.com/images/I/91EbJZH9JWL._SL1500_.jpg", "A movie about a blue person.")
+        var movie10: Movie = Movie("Smurfs", "Fantasy", "Jack Black", "https://upload.wikimedia.org/wikipedia/en/1/11/TheSmurfs2011Poster.jpg", "A movie about a blue person.")
+
+        with(testMovieData) {
+            add(movie1)
+            add(movie2)
+            add(movie3)
+            add(movie4)
+            add(movie5)
+            add(movie6)
+            add(movie7)
+            add(movie8)
+            add(movie9)
+            add(movie10)
+        }
```

Comment:
Good use of Kotlin scoped functions, though I'd go one step further.  Should you be creating/storing data in the Activity?  Probably not.  Refactor this to an external stub class.